scripts/mirror-images/source-images.yaml

```diff
@@ -0,0 +1,46 @@
+gcr.io:
```

Comment:
re: the format and type of this file

I think this is the input file which the mirroring machinery would use to mirror images to?

the content of the input file should be as simple as 2 entries per line, source and destination

each line will represent source and destination of an individual image

$REGISTRY/$IMAGE_PATH:$TAG $REGISTRY/$IMAGE_PATH:$TAG

Reply:
Hello @navidshaikh

Following is some context on the current format of the `source-images.yaml` file.

I'm using [Skopeo](https://github.com/containers/skopeo) for the image syncing across registries (upstream gcr.io & internal ecr-public). And [here is the documented format for image inventory/source file](https://github.com/containers/skopeo/blob/main/docs/skopeo-sync.1.md#yaml-file-content-used-source-for---src-yaml) that Skopeo takes as part of the following command:

```
skopeo sync --src yaml source-images.yaml --dest docker public.ecr.aws/${ECR_REGISTRY_TOKEN}/ --keep-going
```

In our case, we're just using the `source-images.yaml` as an inventory file for the source images that we want to sync from. The destination is remaining same across images as `public.ecr.aws/${ECR_REGISTRY_TOKEN}/<image-getting-mirrored>/`

Reply:
We'll have use case where the destination image tag will be different from source - beside it looks like we referring only registry, image and tag values in the input file.

If this is to get the first iteration done, its ok. Added the suggestion for the probable UX we'll provide to users for entering their mirroing configs.

Reply:
Following the discussion from the Prow Sync-up call, Resolving this in favor of:

I'll work on extending the current image mirroring mechanism (for use-cases like image promotion) in a separate step.